docs/unifiedpush/aerogear-push-android/google-setup.asciidoc

```diff
@@ -1,45 +1,40 @@
 [[google-setup]]
-== Google Related Setup
+== Obtaining Firebase Cloud Messaging Credentials
 
-Before the Android application is able to receive the notifications, you must authenticate yourself with Google as a developer. Luckily this is both easy and free to do, and doesn't require much set-up. Like iOS, you are only able to test out your push notifications in a real device.
+Before the Android application is able to receive the notifications, you must set up access to the Firebase Console. Luckily, this is both easy and free to do, and doesn't require much set-up. As with iOS, you are only able to test out your push notifications in a real device. This section shows you how to obtain the credentials necessary to set up Firebase Cloud Messaging for your app, including:
 
-=== Registering with Google
+* the Server key
+* the Sender ID
+* the `google-services.json` file that needs to be placed into the /app folder of your Android Studio project.
 
-First, we must authenticate with Google. Google offers the Google Cloud Messaging (GCM) platform, formerly known as Cloud to Device Messaging (C2DM). You will need a Google Account for this. Now open the https://cloud.google.com/console[Google Cloud Console].
+=== Project Setup and App Registration using the Firebase Console
 
+First, you must register with the Firebase Cloud Messaging (FCM) platform, formerly known as Google Cloud Messaging (GCM). You need a Google Account to do this. Start by opening the https://console.firebase.google.com[Firebase Console].
```

Comment:
Suggestion: I think "you must" can be omitted. We can simply say _"First, register..."_